backend/dal/Helpers/Constants/ClassificationTypes.cs

```diff
@@ -0,0 +1,21 @@
+namespace Pims.Dal.Helpers.Constants
+{
+    /// <summary>
+    /// Classification Types static class, provides constant values for Classifications.
+    /// </summary>
+    public static class ClassificationTypes
```

Comment:
While having the file isn't bad, we didn't have it because classifications technically come from the database and their `Id` could be anything.  Regrettably due to time we haven't always applied business rules with this in-mind.  Additionally you shouldn't use capitalization in C# in an `enum`.  The normal coding standard for c# is Pascal Case (refer to other enum examples).

Reply:
Ooops totally knew that too, just in TS mode lol. Thanks!

Reply:
Ideally we would use code values instead of ids for everything, but yeah, constants are quick and dirty.